bundles/EcommerceFrameworkBundle/Controller/IndexController.php

```diff
@@ -92,6 +94,10 @@ public function getValuesForFilterFieldAction(Request $request)
                 $data = $helper->getGroupByValuesForFilterGroup($columnGroup, $productList, $request->get('field'));
             }
 
+            $event = new GenericEvent(null, ["data" => $data, "field" => $request->get('field')]);
+            \Pimcore::getEventDispatcher()->dispatch($event, IndexServiceEvents::GET_VALUES_FOR_FILTER_FIELD_PRE_SEND_DATA);
```

Comment:
Please use DI to get the event-dispatcher (type hint).

Reply:
done